installing/installing_vsphere/installing-vsphere-network-customizations.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="installing-vsphere-network-customizations"]
-= Installing a cluster on vSphere with network customizations
+= Installing a cluster on vSphere with network customizations (UPI)
```

Comment:
UPI is not allowed in doc body text.

Reply:
No problem.

I noticed that we have two doc books with the same name:

https://docs.openshift.com/container-platform/4.14/installing/installing_vsphere/installing-vsphere-network-customizations.html[Installing a cluster on vSphere with network customizations]

https://docs.openshift.com/container-platform/4.14/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html[Installing a cluster on vSphere with network customizations]